GNUmakefile

```diff
@@ -38,7 +39,7 @@ attach:
 acceptance: fmtcheck
 	export TF_ACC=1
 	test -n ARTIFACTORY_USERNAME && test -n ARTIFACTORY_PASSWORD && test -n ARTIFACTORY_URL \
-		&& go test -v -parallel 20 ./pkg/...
+		&& go test -ldflags="-X '${PKG_VERSION_PATH}.Version=${NEXT_VERSION}-test'" -v -parallel 20 ./pkg/...
```

Comment:
I assume you mean to remove `-test`

Reply:
No, I add it explicitly. Similar to `debug_install`. I'm ok with removing since it has no effect really :)